Implement override keyword (#7563)
ethereum/solidity: docs/structure-of-a-contract.rst

Function modifiers can be used to amend the semantics of functions in a declarative way
(see :ref:`modifiers` in the contracts section).

> A function modifier must have a unique name. Overloading, that is, having the

Reviewer: Maybe clearer like that
```suggestion
Overloading, that is, having the
```
?

(unique to what?)

        }
    }

> 

Reviewer 2: Perhaps move this section to `inheritance.rst` and link it here? This way, we would keep inheritance specs in one place.

> Function modifiers can override each other. This works similar as function

Reviewer 2: ```suggestion
Function modifiers can override each other. This works similar to function
```

Author: @ChrisChinchilla we need your authority in relation to the English language here. How should this be phrased?

Reviewer 3: @erak was mostly right, but maybe one small thing.

```suggestion
Function modifiers can override each other. This works in a similar way to functions
```

Reviewer: Is it similar or identical?

> overriding. The override keyword must be used in the overriding contract:

Reviewer 2: ```suggestion
overriding. The ``override`` keyword must be used in the overriding contract:
```
